optifit backend/validation.py

```
@@ -88,6 +88,14 @@ def validate_video_file(file):
             f"Unsupported file format '{file_ext}'. "
             f"Allowed formats: {', '.join(ALLOWED_EXTENSIONS)}"
         )
+    #mime type check
+    mime_type, _ = mimetypes.guess_type(file.filename)
+    if not mime_type or not mime_type.startswith('video/'):
+        raise UnsupportedMediaTypeError(f"Invalid MIME type: {mime_type}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)